### DISCUSSION TITLE
Input issues when switching between the OrbStack virtual machine and Mac.

### DISCUSSION THREAD

**Discussion by DimplesY** [ghostty-org/ghostty#4028] [Feature Requests, Ideas] [answered]
When I switch between the OrbStack virtual machine and my Mac, a strange phenomenon occurs, preventing me from inputting normally.

https://github.com/user-attachments/assets/70cc62dd-dd52-438a-b893-eea9c646baf1

**Comment by mitchellh** [comment, marked as answer]
This appears to be a terminfo issue.
